External/FEXCore/unittests/Emitter/System_Tests.cpp

```diff
@@ -0,0 +1,147 @@
+#include "TestDisassembler.h"
+
+#include <catch2/catch.hpp>
+#include <fcntl.h>
+
+static TestDisassembler Emit;
```

Comment:
Instead of having a static global for this, you can turn it into a "Fixture". In this case, that just requires replacing `TEST_CASE(...)` with `TEST_CASE_METHOD(TestDisassembler, ...)`. You can then access TestDisassembler's member functions directly in the test logic, i.e. you can drop the global TestDisassembler and `TEST_SINGLE` simplifies to:
```cpp
#define TEST_SINGLE(emit, expected) \
{ \
  emit; \
  const auto Str = DisassembleSingle(); \
  CHECK(Str == expected); \
}
```

Reply:
Resolved.